src/Hydration.js

```diff
@@ -5,34 +5,36 @@ class Hydration {
   }
 
   calculateToalOunces() {
-    const total = this.userData.reduce(function(sum,current) {
-      return sum + current.numOunces
-  }, 0);
-  return total;
+    const total = this.userData.reduce(function (sum, current) {
+      return sum + current.numOunces;
+    }, 0);
+    return total;
   }
 
   calculateDailyOunces(date) {
-    let thisDay;
- 
-    thisDay = this.userData.filter(element => element.date === date);
-    return thisDay[0].numOunces;
+    const thisDay = this.userData.filter(point => point.date === date);
+    if (thisDay) {
+      return thisDay[0].numOunces;
+    } else {
+      return 'You don\'t have any hydration info from today';
+    }
   }
-  
+
   findDay(date) {
     const day = this.userData.find(dataPoint => dataPoint.date === date);
     return day;
   }
 
   calculateWeeklyOunces(endDate) {
     const day = this.findDay(endDate);
-    const userData = this.userData.reverse()
+    const userData = this.userData.reverse();
     const index = userData.indexOf(day);
     const week = userData.splice(index, 7);
-    const weeklyOunces = week.map(({date, numOunces}) => ({date, numOunces}));
+    const weeklyOunces = week.map(day => day.numOunces).reduce((a, b) => a + b);
```

Comment:
We already talked about this during out stand up today, we may want this to have all the data for the whole week